### ISSUE TITLE
Documentation gaps for configuring JBossMQ with EI

### ISSUE THREAD

**Issue by prabushi** [wso2/product-ei#1368]
**Description:**
Current JMS Listener/Sender configuration does not seem to work as expected. (https://docs.wso2.com/display/EI611/Configure+with+JBossMQ).

Following are configurations that used to locally configure JBoss with EI 6.1.1.

axis2.xml
```
<transportReceiver name="jms" class="org.apache.axis2.transport.jms.JMSListener">
    <parameter name="default" locked="false">
        <parameter name="java.naming.factory.initial">org.jnp.interfaces.NamingContextFactory</parameter>
        <parameter name="java.naming.provider.url">jnp://localhost:1099</parameter>
        <parameter name="java.naming.factory.url.pkgs">org.jboss.naming:org.jnp.interfaces</parameter>
        <parameter name="transport.jms.ConnectionFactoryJNDIName">ConnectionFactory</parameter>
    </parameter>
</transportReceiver>

<transportSender name="jms" class="org.apache.axis2.transport.jms.JMSSender">
   <parameter name="default" locked="false">
      <parameter name="java.naming.factory.initial" locked="false">org.jnp.interfaces.NamingContextFactory</parameter>
      <parameter name="java.naming.provider.url" locked="false">jnp://localhost:1099</parameter>
      <parameter name="transport.jms.ConnectionFactoryJNDIName" locked="false">ConnectionFactory</parameter>
      <parameter name="transport.jms.ConnectionFactoryType" locked="false">queue</parameter>
      <parameter name="transport.jms.Destination">queue/ProducerQueue</parameter>
   </parameter>
</transportSender>
```
For the above configuration, we need to create the ProducerQueue at the JBoss Application Server. For example, we can create sample-queue.xml at ../jboss-5.1.0.GA/server/default/deploy/queue/sample-service.xml
sample-service.xml configuration will be as follows.
```
<server>
  <mbean code="org.jboss.jms.server.destination.QueueService"
         name="jboss.messaging.destination:service=Queue,name=ProducerQueue"
         xmbean-dd="xmdesc/Queue-xmbean.xml">
    <attribute name="JNDIName">queue/ProducerQueue</attribute>
    <depends optional-attribute-name="ServerPeer">jboss.messaging:service=ServerPeer</depends>
    <depends>jboss.messaging:service=PostOffice</depends>
  </mbean>
</server>
```
Further, it would be better to have some examples.
JMS producer example: https://medium.com/@dilsimchandrasena/esb-as-a-jms-producer-sending-messages-to-jbossmq-using-esb-d41609d497be

JMS Listener example:
```
<proxy xmlns="http://ws.apache.org/ns/synapse"
       name="ProducerQueue"
       transports="jms"
       startOnLoad="true">
   <description/>
   <target>
      <inSequence>
         <log level="custom">
            <property name="Message" value="CONSUME MESSAGE !!"/>
         </log>
      </inSequence>
	  <endpoint>
         <address uri="http://172.17.0.1:8280/jboss-queue/log-msg" />
      </endpoint>
   </target>
</proxy>
```

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**
6.2.0

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**
